src/plotting/plotSettings/xAxisTab.component.tsx

```diff
@@ -168,134 +196,125 @@ const XAxisTab = (props: XAxisTabProps) => {
   const xAxisLabel =
     xAxisChannel && xAxisChannel.name ? xAxisChannel.name : XAxis;
 
+  const [fromOpen, setFromOpen] = useState(false);
+  const [toOpen, setToOpen] = useState(false);
+
   return (
     <Grid container spacing={1} mt={1}>
       <Grid container item spacing={1}>
-        <Grid item xs={6}>
-          {XAxisScale === 'time' ? (
-            <LocalizationProvider dateAdapter={AdapterDateFns}>
-              <DateTimePicker
-                inputFormat="yyyy-MM-dd HH:mm"
-                mask="____-__-__ __:__"
-                value={fromDate}
-                maxDateTime={toDate || new Date('2100-01-01 00:00:00')}
-                componentsProps={{
-                  actionBar: { actions: ['clear', 'cancel', 'accept'] },
-                }}
-                onChange={(date) => {
-                  setFromDate(date);
-                }}
-                views={['year', 'month', 'day', 'hours', 'minutes']}
-                OpenPickerButtonProps={{
-                  size: 'small',
-                  'aria-label': 'from, date-time picker',
-                }}
-                renderInput={(renderProps) => {
-                  const error =
-                    // eslint-disable-next-line react/prop-types
-                    (renderProps.error ||
-                      invalidDateRange ||
-                      (fromDate && !isValid(fromDate))) ??
-                    undefined;
-                  let helperText = 'Date-time format: yyyy-MM-dd HH:mm';
-                  if (invalidDateRange) helperText = 'Invalid date-time range';
-
-                  return (
-                    <TextField
-                      {...renderProps}
-                      id="from date-time"
-                      inputProps={{
-                        ...renderProps.inputProps,
-                        style: {
-                          fontSize: 10,
-                        },
+        <ClickAwayListener
+          onClickAway={() => setFromOpen(false)}
+          mouseEvent="onMouseDown"
+        >
+          <Grid item xs={6}>
+            {XAxisScale === 'time' ? (
+              <LocalizationProvider dateAdapter={AdapterDateFns}>
+                <DateTimePicker
+                  format="yyyy-MM-dd HH:mm"
+                  value={fromDate}
+                  maxDateTime={toDate || new Date('2100-01-01 00:00:00')}
+                  onChange={(date) => {
+                    setFromDate(date as Date);
+                  }}
+                  open={fromOpen}
+                  onOpen={() => setFromOpen(true)}
+                  onAccept={() => setFromOpen(false)}
+                  onClose={() => setFromOpen(false)}
+                  closeOnSelect={true}
+                  views={['year', 'month', 'day', 'hours', 'minutes']}
+                  slots={{
+                    textField: CustomTextField,
+                  }}
+                  slotProps={{
+                    actionBar: { actions: ['clear'] },
```

Comment:
This picker only has the clear option whereas the to picker has both clear and today options